.travis.yml

```diff
@@ -67,11 +69,16 @@ matrix:
     env: JOB_NAME=cmake-mingw
   - os: osx
     arch: ppc64le
+  - os: osx
+    arch: s390x
```

Comment:
osx on s390x? Is that a thing?

Reply:
That's actually in the `exclude` section ;-)